### ISSUE TITLE
 Softmax layer is missing in the code.

### ISSUE THREAD

**Issue by Srinidhipy** [gpleiss/efficient_densenet_pytorch#43]
Hi,
I am currently trying to understand ur densenet code. As given in the paper DENSENET which is given as follows:
**At the end of the last dense block, a global average pooling is performed and then a softmax classifier is attached.**
But I am unable to find softmax layer in your optimised code. Please, could you kindly provide me how this has been implemented in Densenet.py file.
Thanks

**Comment by gpleiss** [comment]
The softmax is handled by the `torch.nn.functional.cross_entropy` loss. See https://pytorch.org/docs/stable/nn.html?highlight=cross%20entropy#torch.nn.functional.cross_entropy

**Comment by Srinidhipy** [comment]
Thanks for your kind reply.
